README.md

```diff
@@ -10,9 +26,9 @@ sudo pip install mkdocs-material markdown-include
 mkdocs serve --dev-addr 0.0.0.0:8000
 ```
 
-Deploying to GitHub pages:
+#### Deploying to GitHub pages:
```

Comment:
This section can be removed, since the CI handles deployment and most contributors don't have the rights to deploy the main repo anyways.